galaxy.yml

```
@@ -0,0 +1,63 @@
+---
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: confluent
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: kafka
```

```suggestion
name: platform
```
? @domenicbove WDYT?

---

Yes we call it "confluent platform" with kafka being one component. Go with platform